app/cdap/components/Connections/ImportConnectionBtn/index.tsx

```diff
@@ -85,6 +91,7 @@ export default function ImportConnectionBtn({ onCreate, className = null }) {
         <input
           type="file"
           accept=".json"
+          onClick={handleFileClear}
           onChange={handleFile}
```

Comment:
LGTM
One nitpick is that when we meet `<If>` tag, we could always replace them with {CONDITION_STATEMENT && <>}, just to kill some of the tech debt